www/include/configuration/configObject/meta_service/DB-Func.php

```diff
@@ -49,14 +49,15 @@ function testExistence($name = null)
     if (isset($form)) {
         $id = $form->getSubmitValue('meta_id');
     }
-    $query = "SELECT meta_id FROM meta_service WHERE meta_name = '" . htmlentities($name, ENT_QUOTES, "UTF-8") . "'";
-    $dbResult = $pearDB->query($query);
-    $meta = $dbResult->fetch();
+    $query = "SELECT meta_id FROM meta_service WHERE meta_name = :meta_name";
+    $statement = $pearDB->prepare($query);
+    $statement->bindValue(':meta_name', htmlentities($name, ENT_QUOTES, "UTF-8"));
```

Comment:
missing param type

Reply:
@kduret isn't by default String ? ^^ but i'll add it anyways